README.md

```diff
@@ -1 +1,63 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Description the project.
+This is my portfolio
+
+## Built With
+
+- Major languages
+HTML, CSS
+- Frameworks
+- Technologies used
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
## Optional

- [ ] Please remove unused sections from your README to look more professional. 👍

Reply:
resolved 👍